CONTACT.md

```diff
@@ -0,0 +1,4 @@
+Welcome to the community.
+
+Settle differences by offering and accepting constructive critique, within desired privacy.
+Finding yourselves unable, e-mail Valentin Silvestre (the project maintainer) at vsilvestre.pro@gmail.com
```

Comment:
Please add my adress to this, aswell, Nicolas Mylle, nicolas.mylle.pro@gmail.com